arrays/arrays.c

```diff
@@ -108,14 +108,27 @@ char *arr_read(Array *arr, int index) {
 void arr_insert(Array *arr, char *element, int index) {
 
     // Throw an error if the index is greater than the current count
+    if (index >= arr->count) {
+        printf("Index exceeds the size of the array.");
+        return;
+    }
 
     // Resize the array if the number of elements is over capacity
+    if (arr->count >= arr->capacity) {
+        // resize goes here
+        printf("time to resize yo");
```

Comment:
lol and yes, well handled. you can also `return` right after while you get `resize()` working, so your function doesn't keep going